config/phenomenal_tools2container.yaml

```diff
@@ -394,13 +383,6 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - mzml2isa
-- docker_image_override: isajson-validator
```

Comment:
This goes away as well?

Reply:
Ahh, I see downwards that you are actually removing the wrapper... I thought this was a merging error... please neglect my previous comment...